footer.cfm

```diff
@@ -17,7 +17,7 @@
               <a href="https://www.linkedin.com/company/the-round-league/" target="_blank"><i class="fa fa-linkedin" aria-hidden="true"></i></a>
             </li>
             <li>
-              <a href="https://goo.gl/maps/tUs7Jvvfgn4hi4HY8" target="_blank">4150 SW Watson Ave, Beaverton, OR 97005</a>
+              <a href="https://www.google.com/maps/dir/Current+Location/The+Round+League" target="_blank">4150 SW Watson Ave, Beaverton, OR 97005</a>
```

Comment:
I also can't tell if this works because it does it from my house to round league which is the original bug lol. Can we replace it with this link tho?

https://maps.app.goo.gl/1vBkBnVL7wMbAps2A

Reply:
@Jwwason